doc/integration.md

```diff
@@ -109,6 +109,8 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         
         // Here we store the token assigned to the user
```

Comment:
This comment used to talk about both lines below it. Change "store" to "retrieve".